frontend/src/components/ModalProfile.tsx

```diff
@@ -25,15 +25,66 @@ export const ModalProfile:React.FC<ModalProfile> = ({bio, isExpanded}) => {
+                <p className='modal-about-text'>{ele}</p>
+            ))}
+            
+            <div className="modal-flex-div">
+                <div className="modal-flex-section">
+                <h2>Experience</h2>
+                    {bio.experience.map((ele, index) => (
+                        <div className="modal-general-div">
+                            <p className='modal-med-heading' key={index}>{ele.company}</p>
+                            <p className='modal-sm-text'>{ele.title}<span className='modal-sm-text-italic'>{ele.date}</span></p>
+                        </div>
+                    ))}
+                </div>
+                <div className="modal-flex-section">
+                    <h2>Skills</h2>
+                    <h5>Programming Languages</h5>
+                        <div className='modal-skills-div'> 
+                            {bio.skills.languages.map((ele, index) => (
+                                    <p key={index} className='modal-skills-text'>{ele}</p>
+                            ))}       
+                        </div>
+                    <h5>Frameworks & Libraries</h5>
+                        <div className='modal-skills-div'> 
+                            {bio.skills.libraries.map((ele, index) => (
+                                    <p key={index} className='modal-skills-text'>{ele}</p>
+                            ))}       
+                        </div>
+                    <h5>Databases</h5>
+                        <div className='modal-skills-div'> 
+                            {bio.skills.databases.map((ele, index) => (
+                                    <p key={index} className='modal-skills-text'>{ele}</p>
+                            ))}       
+                        </div>
+                    <h5>Other Tools & Technologies</h5>
+                        <div className='modal-skills-div'> 
+                            {bio.skills.other.map((ele, index) => (
+                                    <p key={index} className='modal-skills-text'>{ele}</p>
+                            ))}       
+                        </div>
+                </div>
+            </div>
+            
             <h2>Education</h2>
-            <p>{bio.education}</p>
-            <h2>Links</h2>
-            <p>{bio.links}</p>
+                {bio.education.map((ele, index) => (
+                    <div className="modal-general-div">
+                        <p className='modal-med-heading' key={index}>{ele.school}</p>
+                        <p className='modal-sm-text' key={index}>{ele.schoolInfo}<span className='modal-sm-text-italic'>{ele.schoolDate}</span></p>
+                    </div>
+                ))}
+            
+            <h2>Let's Connect!</h2>
+            <p className='modal-med-heading'>{bio.email}</p>
+            <div className='modal-socials'>
+                {bio.socials.map((ele, index) => (
+                    <a href={ele.url} target='blank'>
```

Comment:
if we update to:

<a href={ele.url} target={`_blank_{${index}`} rel="noopener noreferrer" key={index}>

it will open the LinkedIn/GitHub links in their own windows

note: the highlighted section is encased in back-ticks, GitHub won't let me paste it without marking that one section as "code"

Reply:
Fixed!